poly-markdown.el

```diff
@@ -62,7 +62,7 @@
   :tail-matcher (pm-make-text-property-matcher 'markdown-yaml-metadata-end))
 
 (define-auto-innermode poly-markdown-fenced-code-innermode poly-markdown-root-innermode
-  :head-matcher (cons "^[ \t]*\\(```{?[[:alpha:]].*\n\\)" 1)
+  :head-matcher (cons "^[ \t]*\\(```[ \t]*{?[[:alpha:]].*\n\\)" 1)
```

Comment:
Is this of any practical importance? I think it could be left as it is without any loss of generality, no? What's the point of that space in gfm?

Reply:
Because that's the standard way to write Github flavored markdown?

I've taken a look around of README in github, most of them have space separator between ` ``` ` and language name.
For example this [README.md](https://raw.githubusercontent.com/PowerShell/PSScriptAnalyzer/master/README.md)

Also, if you're using `gfm-mode`, it automatically insert a space between code opening block and code name.
For example:
![67ee48f7-042d-467d-8431-8bb745d4b277](https://user-images.githubusercontent.com/2631472/61196117-66a1cc00-a707-11e9-829c-2a9ae365ccef.gif)

You can see that a space is inserted automatically.
Without changing the regex, I will have to delete that space every time, and it's a big annoyance.

I think that it would also benefit `markdown-mode` to be able to recognize that space between opening code block and language name too.

Reply:
I see. Actually the mode matcher already takes into account the spaces. So should be fine indeed.